elements/pl-multiple-choice/pl-multiple-choice.py

```diff
@@ -185,8 +187,8 @@ def prepare(element_html, data):
     # overwriting previous choice(s)
     display_answers = []
     correct_answer = None
-    for (i, (index, correct, html)) in enumerate(sampled_answers):
-        keyed_answer = {'key': pl.index2key(i), 'html': html}
+    for (i, (index, correct, html, hint)) in enumerate(sampled_answers):
```

Comment:
Since you're changing this tuple to have 4 elements, you also need to update other places that handle this tuple, like lines 51 and 54, which get correct/incorrect answers from JSON files.

Reply:
And lines 174 and 182, for all-of-these/none-of-these.

Reply:
Alright, I'll put `None` into the JSON file tuples in lines 51 and 54 for now since I'm not sure how hints will work with them at the moment, and I'll add something similar to the `none-of-the-above-hint` attribute you talked about which will be inserted into the tuple in lines 174 and 182.